doc/source/whatsnew/v0.19.1.txt

```diff
@@ -30,3 +30,5 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
+
+- Bug in date range treating e.g. 0.5s as 5s (:issue:`8419`)
```

Comment:
can you add occurrence condition, like

- Bug in multiple offset aliases with decimal points is regarded as int (e.g. 0.5s as 5s).